DPGAnalysis/HcalTools/plugins/CMTRawAnalyzer.cc

```diff
@@ -8821,23 +8818,23 @@ void CMTRawAnalyzer::fillDigiAmplitude(HBHEDigiCollection::const_iterator& digiI
   pedestalaver4 /= c4;
   pedestalwaver9 = sqrt(pedestalwaver9 / TSsize);
   pedestalwaver4 = sqrt(pedestalwaver4 / c4);
-  if (ts_with_max_signal > -1 && ts_with_max_signal < 10)
+  if (ts_with_max_signal > 0 && ts_with_max_signal < TSsize)
```

Comment:
Why here and in the following you have "> 0" and you have "> -1" in all other cases?

Reply:
in principle, it does not matter, but -1 is more correct, Thank you !